deploy/charts/emqx-operator/values.yaml

```diff
@@ -12,6 +12,11 @@ image:
   # Overrides the image tag whose default is the chart appVersion.
   tag: ""
 
+kubeRbacProxy:
```

Comment:
Maybe `kube-rbac-proxy` is better, I'm not sure, but cert manager use `cert-manager`, not `certManager` in this file

Reply:
Done,update `kubeRbacProxy` -> `kube-rbac-proxy`